certbot/tests/client_test.py

```diff
@@ -178,8 +179,7 @@ def test_obtain_certificate_from_csr(self, mock_logger):
             test_csr)
         mock_logger.warning.assert_called_once_with(mock.ANY)
 
-    @mock.patch("certbot.client.crypto_util")
-    def test_obtain_certificate(self, mock_crypto_util):
+    def _test_obtain_certificate(self, mock_crypto_util, key_to_check=None):
```

Comment:
This can probably be refactored further to use with other test_obtain_cert.* methods, but I will leave it for now if there are no objections